src/resource/jsonschema/__tests__/fixtures.js

```diff
@@ -51,21 +51,19 @@ export const NestedReferenceSchema = JSONSchemaResource.all({
     id: 'NestedReference',
     properties: {
         value: {
-            // XXX $ref: ChildSchema.toRef(),
-            $ref: '#Child',
+            $ref: ChildSchema.toRef(),
```

Comment:
Our API is now much easier as well. If you want a reference, just use `Resource.toRef()`.

It's no longer possible to call `addReference`